python/lsst/ts/ess/csc/rpi_data_client.py

```diff
@@ -167,6 +288,14 @@ def get_config_schema(cls) -> Dict[str, Any]:
             can be used to give the location of each probe. In that case the
             locations should be given in the order of the channels.
           type: string
+        num_samples:
+          description: >-
+            Number of samples per telemetry sample. Only relevant for
+            certain kinds of data, such as wind speed and direction.
+            Ignored for other kinds of data.
+          type: integer
+          default: 25
```

Comment:
I'd prefer to have this default to 0, like `num_channels`, so any device needing it needs to explicitly set it.

Reply:
How about making it optional, with no default. It will be None for most rules.

Reply:
Great, thanks!